Update ms-Oriya.js

durga598 opened the pull request:
updated odia letter Ja and danda.

Mottie commented:
Hi @durga598!

Thanks for your help! Would you also look at the top row, I noticed two `0`'s in a row. Please help me fix that as well.

durga598 commented:
@Mottie, updated extra 0 with a minus sign and added some others as given in this [odia layout](https://docs.microsoft.com/en-us/globalization/keyboards/kbdinori.html)

Mottie commented:
Awesome thanks!

While I have you helping, do you think you could create an `or.js` language file as well? You can use [this template](https://github.com/Mottie/Keyboard/blob/master/languages/_language_template.js) file. No pressure.
